src/rules/NoIgnoredReturnRule.ts

```diff
@@ -0,0 +1,240 @@
+import * as Lint from "tslint";
+import { IOptions, IRuleMetadata, Utils } from "tslint";
```

Comment:
it already exists https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md